docs/.Doxyfile

```diff
@@ -378,7 +378,7 @@ TOC_INCLUDE_HEADINGS   = 0
 # The default value is: DOXYGEN.
 # This tag requires that the tag MARKDOWN_SUPPORT is set to YES.
 
-MARKDOWN_ID_STYLE      = DOXYGEN
+MARKDOWN_ID_STYLE      = GITHUB
```

Comment:
Why is this necessary/related to a `libjade` integration?

Reply:
It isn't. I likely made this change while debugging documentation generation and this option fixes some issues I was facing with GH style anchors.